app/models/server/models/Users.js

```diff
@@ -1551,6 +1551,17 @@ Find users to send a message by email if:
 
 		return this.find(query, options);
 	}
+
+	getLanguages() {
+		const query = {
+			language: {
+				$exists: true,
+				$ne: '',
+			},
+		};
+
+		return this.find(query);
```

Comment:
This will return all data about the users that match the query but you only need the language field.

I'd either suggest making this as an aggregate to return the count from the database per language rather than return all data to count on the server-side.